tests/test_telemetry.py

```diff
@@ -485,3 +485,30 @@ def test_context_contains_os():
     context.pop("os")
 
     assert "os" in telemetry._default_context_fields()
+
+
+def test_context_contains_license_hash(monkeypatch: MonkeyPatch):
+    mock = MagicMock()
+    telemetry.TELEMETRY_CONTEXT = None # make sure we don't use the cached value
```

Comment:
Is this because TELEMETRY_CONTEXT gets set in other unit tests?

Reply:
Yea it gets set in `test_context_contains_os` and does not get updated in these 2 tests (because it's not `None`), so I had to set it to the default value

Reply:
I see, would it be beneficial to reset it in a parameterized pytest fixture (`@pytest.fixture(autouse=True)`), then you don't need to duplicate the line and the TELEMETRY_CONTEXT is always None at the start of every unit test?

Reply:
Or would that cause other tests to fail (although that would be a bad sign since unit tests shouldn't be dependent)?

Reply:
Added the `@pytest.fixture(autouse=True)` to avoid test conflicting